docs/docs/Development.md

```diff
@@ -0,0 +1,106 @@
+---
+title: "Development"
+linkTitle: "Development"
+weight: 3
+description: >
+  How to develop Triggers and Functions
+---
+
+To aid in the development of new libraries of Triggers and Functions you can use the type declaration files for the Triggers and Functions API. This can be used by your preferred development environment to provide autocomplete and type check based on the type information. You can install it using the following command:
```

Comment:
```suggestion
To aid in the development of new libraries of Triggers and Functions you can use the type declaration files for the Triggers and Functions API. This can be used by your preferred development environment to provide autocompletion and type checking based on the type information. You can install it using the following command:
```